### PR TITLE
Cloud changelog - Oct 13th

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -4,12 +4,45 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
 - [Release 2022-08-25](#release-2022-08-25)
-- [Release 2022-08-18](#release-2022-08-18)
+
+## Release 2022-10-13
+
+### Improvements
+
+#### User Interface (UI)
+ - Insights now filters out posts made by plugins and OAuth apps.
+ - Added a “Last active” status to the profile popover and to the **Direct Message** channel header to be able to see when a user was last online. This status only displays for users who are Away, Offline or DND.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:20]:
```suggestion
 - Added a “Last active” status to the profile popover and to the **Direct Message** channel header that indicates when a user was last online. This status only displays for users who are Away, Offline, or DND.
```

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -4,12 +4,45 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added a shortcut ``Ctrl/Cmd + Shift + U`` to filter channels by unread.
+ - The default number of **Direct Message** channels shown in the sidebar is now 20 to 40.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:22]: the new default is 20 _from_ 40?

File: source/install/cloud-changelog.md [hunk 3]
```diff
@@ -4,12 +4,45 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Updated the company name in the **About Modal** to use the company name of the cloud customer instead of the company name in the cloud license.
+ - Downgraded Brazilian Portuguese and Romanian language support to Alpha.
+
+### Bug Fixes
+ - Fixed an issue where a randomly generated default message-ID was not added for every outgoing email.
+ - Fixed an issue where custom groups could be created with at-mention names that are reserved words (@channel, @here, @all).
+ - Fixed an issue where 404 errors were shown when APIv4 had an incorrect content-type header.
+ - Fixed an issue where messages from bot and webhook could not be forwarded.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:30]: would it make sense to pluralize "bot" and "webhook" as: "Fixed an issue where messages from **bots** and **webhooks** could not be forwarded."

File: source/install/cloud-changelog.md [hunk 4]
```diff
@@ -4,12 +4,45 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Fixed an issue where the plugin limit read "closer" even when the full limit of 5 plugins was enabled.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:31]: I'm not 100% clear on what this bullet point is communicating -- specifically via the term "closer"

File: source/install/cloud-changelog.md [hunk 5]
```diff
@@ -4,12 +4,45 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Fixed an issue where inline images did not appear in the channel header.
+ - Fixed an issue with the emoji skin tone selector animation.
+ - Fixed an issue where an archived root post with a file errored out when trying to view its channel.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:34]: "errored out" may not be a phrase that's widely understood. 

Would the following be technically accurate and clearer?  "Fixed an issue where attempting view a channel that contains an archived root post and a file attachment displayed an error."

**Review by cwarnermm — COMMENTED** [review]
Thanks! Feedback provided inline.

File: source/install/cloud-changelog.md (hunk 2 above)
Comment (esethna) [reply source/install/cloud-changelog.md:22]: new is 40

**Review by esethna — APPROVED** [review]

File: source/install/cloud-changelog.md (hunk 4 above)
Comment (amyblais) [reply source/install/cloud-changelog.md]: Clarified

**Review by marianunez — APPROVED** [review]

**Review by johndavidlugtu — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 6]
```diff
@@ -4,12 +4,63 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Fixed an issue where the screen reader did not announce a successful login when logging in.
+ - Fixed a few broken links at **System Console > User Management > Permission Schemes**.
+ - Fixed an issue where users were able to forward messages to users who are deactivated.
+ - Fixed an issue where "Threads" were not shown in the unread filter view even if there weren't unread threads.
+ - Fixed an issue where the user’s full name was not shown when adding people to a channel via the ``Add people`` modal.
+ - Reverted the new search of names in PostgreSQL using full text search introduced in v7.3.0 due to a performance regression.
+
+### Known Issues
+ - "More" menu for Pinned posts on the right-hand side is cut-off [MM-46987](https://mattermost.atlassian.net/browse/MM-46987).
+ - Board Admins cannot add other non-board members as "Admins" via autocomplete list. As a workaround, admins can add board members via the Share dialog and set the user's role to "Admin". See the [GitHub issue #3957](https://github.com/mattermost/focalboard/issues/3957) for more information.
```
Comment (wuwinson) [inline source/install/cloud-changelog.md:62]:
```suggestion
```

**Review by gabrieljackson — APPROVED** [review]

**Pull request merged**